src/ServicePulse.Host/vue/src/resources/Endpoint.ts

```diff
@@ -0,0 +1,96 @@
+export interface Endpoint {
+  name: string;
+  errorCount: number;
+  serviceControlId: string;
+  isScMonitoringDisconnected: boolean;
+  metrics: EndpointMetrics;
+  isStale: boolean;
+  endpointInstanceIds: string[];
+  disconnectedCount: number;
+  connectedCount: number;
+}
+
+export interface EndpointValues {
+  points: number[];
+  average: number;
+}
+
+export interface EndpointValuesWithTime extends EndpointValues {
+  timeAxisValues: Date[];
+}
+
+export interface EndpointMetrics {
+  [index: string]: EndpointValues;
+  queueLength: EndpointValues;
+  throughput: EndpointValues;
+  retries: EndpointValues;
+  processingTime: EndpointValuesWithTime;
+  criticalTime: EndpointValuesWithTime;
+}
+
+const defaultMetricData: EndpointValues = {
```

Comment:
These defaults don't feel like belong here

Reply:
where else? they're co-located with the exported function that uses them

Reply:
The plan was only to have types that are "copies" from the REST API c# code types in the resources folder.
This does not seem to be a type from the backend, hence I think we can move these somewhere else.